protractor/local.config.ts

```diff
@@ -3,7 +3,8 @@ import { reporter } from './helpers/reporter';
 
 export const config: Config = {
   framework: 'jasmine',
-  specs: ['../test/google.spec.js'],
+  specs: ['../test/**/*.spec.js'],
+  getPageTimeout: 2000,
```

Comment:
Why is this timeout different to the headless one?

Reply:
I thought it might be different, GUI timeout might take longer than headless due to component rendering time, is this wrong?

Reply:
both are rendering elements. However, using the local config you will never see the UI bc it is usually running in a non-graphic machine...You could use services like saucelabs to improve your tests